helix-term/src/commands.rs

```diff
@@ -133,7 +133,7 @@ pub struct Command {
 }
 
 macro_rules! commands {
-    ( $($name:ident, $doc:literal),* ) => {
+    ( $($name:ident, $doc:literal),* $(,)? ) => {
```

Comment:
I think you can just force this.
```suggestion
    ( $($name:ident, $doc:literal),* , ) => {
```

Reply:
If we want to enforce it, we could also just move the comma inside the parentheses:
```rs
    ( $($name:ident, $doc:literal,)* ) => {
```